format-specs/geoparquet.md

```diff
@@ -181,14 +181,52 @@ It is RECOMMENDED to always set the orientation (to counterclockwise) if `edges`
 
 #### edges
 
-This attribute indicates how to interpret the edges of the geometries: whether the line between two points is a straight cartesian line or the shortest line on the sphere (geodesic line). Available values are:
-- `"planar"`: use a flat cartesian coordinate system.
-- `"spherical"`: use a spherical coordinate system and radius derived from the spheroid defined by the coordinate reference system.
+This attribute describes describing the interpretation of edges between explicitly
+defined vertices.
+
+- `"planar"`: edges will be interpreted following the language of
+  [Simple features access](https://www.opengeospatial.org/standards/sfa):
+
+  > **simple feature** feature with all geometric attributes described piecewise
+  > by straight line or planar interpolation between sets of points (Section 4.19).
+
+- `"spherical"`: Edges follow the shortest distance between vertices approximated
```

Comment:
Can one describe the `perfect sphere` via a geodetic CRS? That way, we can merge `spherical` and `geodesic` together and recommend everyone to look into the `crs` key for interpreting the edge?

Reply:
yes, a sphere is described by an ellipsoid with an inverse flattening conventionally set to 0 (it should actually be infinity, but the 0 convention is heavily used)
I believe an issue though (mentionned in some other discussion) is that in some situations some software might apply the formulas for spheres on non-spherical ellipsoids as a simplification avoiding using elliptical integrals

Reply:
Now I understand. `spherical` mode actually overrides the edge specified in the `crs`. That's where the confusion came from.

I think this is making great progress. I will schedule another meeting among all parties to settle down on the geography type of Iceberg